webpack-test.config.js

```diff
@@ -0,0 +1,52 @@
+const webpack = require( 'webpack' );
```

Comment:
ok I'm keeping it.

why?

let's just say you change bundler,
- then you will have to remove all that config which "should be in karma opinion"

all that config is related just to the webpack-karma plugin, has nothing specific in common with karma as a whole